common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerRequest.java

```diff
@@ -266,4 +265,10 @@ private static final class SerializedNames {
      */
     @SerializedName(SerializedNames.SUPPRESS_ACCOUNT_PICKER)
     private boolean mSuppressAccountPicker;
+
+    /**
+     * List of browsers that are safe to use for the request.
+     */
+    @SerializedName(SerializedNames.BROWSER_SAFE_LIST)
+    private List<BrowserDescriptor> mBrowserSafeList;
```

Comment:
We discussed offline that we don't need to take this from the client and that Broker should internally limit it to Chrome and Edge.